recipes/datastax.rb

```diff
@@ -35,8 +35,8 @@
   action :install
 end
 
-package node['cassandra']['dsc_version'] do
-  action :install
+execute "install_cassandra" do
+  command "apt-get install -y #{node[:cassandra][:dsc_version]}=#{node[:cassandra][:version]}* cassandra=#{node[:cassandra][:version]}"
```

Comment:
After looking this over again and reviewing the OpsWorks environment, it looks like this recipe isn't being used (and we don't want to use it). With that in mind, what is the relevance of this edit?